src/include/optimizer/stats/stats_util.h

```diff
@@ -0,0 +1,47 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// stats_util.h
+//
+// Identification: src/include/optimizer/stats/stats_util.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <limits>
+
+#include "common/macros.h"
+#include "type/types.h"
+#include "type/value.h"
+#include "type/value_factory.h"
+
+namespace peloton {
+namespace optimizer {
+
+class StatsUtil {
+public:
+  // Convert numeric peloton value type to primitive value.
+  // Return NaN if value is not numeric.
```

Comment:
This looks like a generic function and not specific to StatsUtil. In that case wouldn't it make more sense to make this function a part of the value / type.cpp files ? Since all related such functionalities are present in those files/directories